PathKit/Sources/PathKit/Vector.swift

```diff
@@ -50,12 +50,21 @@ extension Vector {
         self.dy = Double(dy)
     }
 
-    /// Creates a vector pointing to `point`.
+    /// Creates a vector pointing from the origin to `point`.
     ///
     /// - Parameter point: The point used to create the vector.
     public init(_ point: Point) {
```

Comment:
Not a fan of this either.

Reply:
Would you just completely remove this?

Reply:
I would give it a better name. For instance `point.vector(from: .origin)`

Reply:
Can also just do Vector(head: point, tail: origin)